summary: age buckets added (#193)
tarantool/metrics: doc/monitoring/api_reference.rst

        usage of resources (memory and cpu) to calculate the summary.

    :param table params: Table of summary parameters, used for configuring sliding
        window of time. 'Sliding window' consists of several buckets to store observations.

>         New observation are added to each bucket. After a time period, 'head' bucket

Reviewer: ```suggestion
        New observations are added to each bucket. After a time period, 'head' bucket
```

        (bucket from which observations are collected) is reset and next bucket becomes

>         new 'head'. I.e. each bucket will be store observations for the last

Reviewer: ```suggestion
        new 'head'. I.e. each bucket will store observations for
```

        ``max_age_time * age_buckets_count`` seconds before it will be reset.

>         ``max_age_time`` set the duration of each bucket lifetime, i.e., how long

Reviewer: ```suggestion
        ``max_age_time`` sets the duration of each bucket lifetime, i.e., how long
```

        observations are kept before they are discarded, in seconds

>         ``age_buckets_count`` set the number of buckets of the time window. It

Reviewer: ```suggestion
        ``age_buckets_count`` sets the number of buckets of the time window. It
```

    local metrics = require('metrics')

>     -- create a summary with a sliding wingow with 5 age buckets and 60s bucket lifetime

Reviewer 2: ```suggestion
    -- create a summary with a window of 5 age buckets and 60s bucket lifetime
```
